mocks/error.go

```diff
@@ -19,14 +17,10 @@ func (e *Error) Error() string {
 type RequestConstraintError struct {
 	error
 	Constraint verifier
-	Request    *http.Request
+	Request    []byte
```

Comment:
Maybe rename this field since it's no longer a request struct? Think it could be called like `requestDump`.

Reply:
fixed